env.default

```diff
@@ -88,3 +88,9 @@ SENTRY_ENVIRONMENT=
 
 # CDN configuration
 STATIC_HOST=
+
+# Comma separated list of domains to be exempted from rel="nofollow" attribute.
+# e.g. 'mozilla.org,mozilla.design' will match 
+# 'anything.mozilla.org/path' or 'mozilla.design.anything/path'
+# Or alternatively set WHITELISTED_LINKS="*" to exempt all links.
+WHITELISTED_LINKS="mozilla.org,mozilla.design"
```

Comment:
Do you think `WHITELISTED_EXTERNAL_LINKS` would be more descriptive?

Reply:
Thank you! I've renamed it now.